yarn-error.log

```diff
@@ -0,0 +1,3424 @@
+Arguments: 
```

Comment:
we might want to add this file to the gitignore

Reply:
Added to .gitignore and removed the file - thx!